src/type-translator.ts

```diff
@@ -37,13 +38,40 @@ export function typeToDebugString(type: ts.Type): string {
   let debugString = `flags:0x${type.flags.toString(16)}`;
 
   const basicTypes: ts.TypeFlags[] = [
-    ts.TypeFlags.Any,           ts.TypeFlags.String,         ts.TypeFlags.Number,
-    ts.TypeFlags.Boolean,       ts.TypeFlags.Enum,           ts.TypeFlags.StringLiteral,
-    ts.TypeFlags.NumberLiteral, ts.TypeFlags.BooleanLiteral, ts.TypeFlags.EnumLiteral,
-    ts.TypeFlags.ESSymbol,      ts.TypeFlags.Void,           ts.TypeFlags.Undefined,
-    ts.TypeFlags.Null,          ts.TypeFlags.Never,          ts.TypeFlags.TypeParameter,
-    ts.TypeFlags.Object,        ts.TypeFlags.Union,          ts.TypeFlags.Intersection,
-    ts.TypeFlags.Index,         ts.TypeFlags.IndexedAccess,
+    ts.TypeFlags.Any,
+    ts.TypeFlags.String,
+    ts.TypeFlags.Number,
+    ts.TypeFlags.Boolean,
+    ts.TypeFlags.Enum,
+    ts.TypeFlags.StringLiteral,
+    ts.TypeFlags.NumberLiteral,
+    ts.TypeFlags.BooleanLiteral,
+    ts.TypeFlags.EnumLiteral,
+    ts.TypeFlags.ESSymbol,
+    ts.TypeFlags.Void,
+    ts.TypeFlags.Undefined,
+    ts.TypeFlags.Null,
+    ts.TypeFlags.Never,
+    ts.TypeFlags.TypeParameter,
+    ts.TypeFlags.Object,
+    ts.TypeFlags.Union,
+    ts.TypeFlags.Intersection,
+    ts.TypeFlags.Index,
+    ts.TypeFlags.IndexedAccess,
+    ts.TypeFlags.NonPrimitive,
+    ts.TypeFlags.Literal,
+    ts.TypeFlags.StringOrNumberLiteral,
+    ts.TypeFlags.PossiblyFalsy,
+    ts.TypeFlags.StringLike,
+    ts.TypeFlags.NumberLike,
+    ts.TypeFlags.BooleanLike,
+    ts.TypeFlags.EnumLike,
+    ts.TypeFlags.UnionOrIntersection,
+    ts.TypeFlags.StructuredType,
+    ts.TypeFlags.StructuredOrTypeVariable,
+    ts.TypeFlags.TypeVariable,
+    ts.TypeFlags.Narrowable,
+    ts.TypeFlags.NotUnionOrUnit,
```

Comment:
OOC, whats the clang-format logic here. I feel it keeps swapping back and forth.